Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.1_README.md

```diff
@@ -1,29 +1,39 @@
-Enriches accounts using one or more integrations.
-
+Enrich accounts using one or more integrations.
 Supported integrations:
 - Active Directory
+- SailPoint
+- PingOne
+- Okta
```

Comment:
Okta and AWS are already in the Account enrichment -IAM subplaybook which the subplaybook is also used in that playbook. Why do we need it twice?

Reply:
Also, AD in both playbooks.

Reply:
AWS and OKTA has 2 integrations - with 2 different commands to retrieve account information. Hence, there is a separation for that.
<img width="649" alt="image" src="https://user-images.githubusercontent.com/88268646/185324043-fa6e41a0-aaf5-4858-b59b-b68f6a52f277.png">
<img width="676" alt="image" src="https://user-images.githubusercontent.com/88268646/185325785-1726befc-1efe-4359-81a3-d747bf868abc.png">


Now, AD integration contains those 2 commands ( one of them is the generic one `iam-get-user`) inside it so it can be logical to split those playbooks as the commands are not the same (so if someone would like to use only the IAM command - he has a dedicated playbook for that). 

Moreover, the reason for creating the generic command as a sub-playbook -> is the looping ability, to support multiple values for the enrichment process.

Reply:
I forgot to mention that the `iam-get-user` command contains a bit more shrunk output than the dedicated command ( however, it is easier for me to work with the structure across other integrations).
Hence, I thought it might be a good idea to use it as a sub-playbook dedicated to this generic command.